controllers/vaultdynamicsecret_controller.go

```diff
@@ -390,6 +390,7 @@ func (r *VaultDynamicSecretReconciler) revokeLease(ctx context.Context, o *secre
 	}
 	if _, err = c.Write(ctx, "/sys/leases/revoke", map[string]interface{}{
 		"lease_id": leaseID,
+		"sync":     true,
```

Comment:
I think we may want to make this a configurable, with the default of false, since it may be put extra load on Vault when we have a lot of revocations.

Reply:
Removed.